pkg/minikube/image/cache.go

```diff
@@ -31,9 +31,22 @@ import (
 	"k8s.io/klog/v2"
 	"k8s.io/minikube/pkg/minikube/constants"
 	"k8s.io/minikube/pkg/minikube/localpath"
+	"k8s.io/minikube/pkg/minikube/out"
 	"k8s.io/minikube/pkg/util/lock"
 )
 
+// FailFastError type is an error that could not be solved by trying again
+type FailFastError struct {
```

Comment:
this is doesnt need to be called fail fast error, this is about cache, that made sense in the conext of oci

Reply:
also this is NOT used anywhere outside, so please make it "private"

Reply:
Thank you. Got it.